setup.py

```diff
@@ -29,6 +29,7 @@ def get_version(filename):
         'requests >= 2.0',
         'setuptools',
         'tornado >= 4.4, < 5',  # Tornado 5 requires Python >= 2.7.9
+        'bcrypt'
```

Comment:
Why not use https://docs.python.org/3/library/secrets.html#secrets.token_hex instead of a new dep? Or a 2.7 stdlib based equivalent (https://github.com/python/cpython/blob/master/Lib/secrets.py#L60 or so).

Reply:
More specifically `binascii.hexlify(os.urandom(32))`, and then we can leave a TODO about switching to secrets when we are switched to PY3.